src/converters/cover.ts

```diff
@@ -27,12 +27,28 @@ export class CoverCreator implements ServiceCreator {
           exposesHasFeatures(e) &&
           !accessory.isServiceHandlerIdKnown(CoverHandler.generateIdentifier(e.endpoint))
       )
-      .forEach((e) => this.createService(e as ExposesEntryWithFeatures, accessory));
+      .forEach((e) => {
+        let exposesMotorState: boolean = false;
+        const motorStateExpose = exposes.find(
+          (m) =>
+            CoverHandler.generateIdentifier(m.endpoint) === CoverHandler.generateIdentifier(e.endpoint) &&
```

Comment:
Just comparing the `endpoint` should give the same result without doing string manipulation first.

You can also use `groupByEndpoint` instead (from `src/helpers.ts`).